docker-compose-prod.yml

```diff
@@ -18,15 +18,9 @@ services:
 
   neo4j:
     restart: unless-stopped
-    container_name: neo4j
-    build: ./neo4j
-    command: neo4j
-    ports:
-      - "7687:7687"
 
   api:
     restart: unless-stopped
-    container_name: api
     build:
       context: ./api
       dockerfile: ./Dockerfile.prod
```

Comment:
Unless there are plans for different dockerfiles, maybe this can be moved to the base `docker-compose` .